.github/workflows/run_detekt_and_unit_tests.yml

```
@@ -0,0 +1,38 @@
+name: Run Detekt and unit tests
+
+on: push
```

```suggestion
on:
  pull_request:
    types: [ opened, synchronize, edited, reopened ]
  push:
    branches-ignore:
      - develop
      - main
      - 'release/**'
```

Let's update this to run on every PRs. And we can ignore for `develop, main, release/**` branch when we merge to them. 🙏🏻